tools/paket2bazel/Gen.fs

```diff
@@ -40,8 +40,17 @@ let generateTarget (group: Group) (repoName: string) =
                 else
                     s.Substring(0, s.Length - 2))
 
+        let sources =
+            package.sources
+            |> Seq.fold (fun state current -> state + $"\"{current}\", ") ""
+            |> (fun s ->
+                if String.IsNullOrEmpty(s) then
+                    s
+                else
+                    s.Substring(0, s.Length - 2))
+
         sb.Append(
-            $"{i}        (\"{package.name}\", \"{package.version}\", \"{package.sha512sri}\", [{packageDeps}], [{overrides}]),\n"
+            $"{i}        (\"{package.name}\", \"{package.version}\", \"{package.sha512sri}\", [{sources}], None, [{packageDeps}], [{overrides}]),\n"
```

Comment:
How about exposing a flag to set the specific netrc file path? It could be more friendly than generating `None` to save an extra step updating the generated packages.

Reply:
Added a `--netrc-file-label` to the paket2bazel tool. You can now run it with `--netrc-file-label //some:label` and the netrc file will be added to the generated nuget_repo.